test/controllers/EventsControllerSpec.scala

```diff
@@ -55,19 +55,21 @@ class EventsControllerSpec extends UnitWithAppSpec {
     }
 
     "return OK with all events" in new TestFixture {
```

Comment:
This test should be renamed perhaps as "return OK and empty event list when there are no events" or something similar?